testing/btest/scripts/base/utils/json-roundtrip.test

```diff
@@ -0,0 +1,102 @@
+# @TEST-DOCS: Test round-trip JSON encoding and decoding using the Zeek methods
```

Comment:
Thanks for adding this! Could also see this one in `bifs/`, next to the existing `from_json.zeek` test. I thought `base/utils` is used for what's actually located in `base/utils`.

Reply:
There's also `scripts/base/utils/json.test`

Reply:
Fair! Resolved it.